docs/features.rst

```diff
@@ -194,6 +194,14 @@ The key content attributes are not supposed to be hard-coded in
 the Flexible Metadata Format but freely configurable. Multiple key
 content attributes (e.g. script & backend) could be used as well.
 
+Sometimes it is necessary to select node from the metadata tree
+even though it is not leaf. For example, when virtual tests are
+created from a parent test but one wants to keep the parent available
+as a test as well. To do so, one can set fhe following directive::
+
+    /:
+        make-leaf: true
```

Comment:
This approach seems a bit one-use-case focused. What about making the syntax a little bit more flexible and extensible? It could look like this:

    /:
        kind: leaf

This new directive would denote `kind` of the node, or `type`, or some better name if we find one. By default leaves and branches would be detected from the presence of children but it could be explicitly overridden by user if desired. Possible values could be:

* branch
* leaf
* hidden

When `climb()`ing the tree, it would be possible to select whether leaves or branches should be yielded and whether hidden nodes should be ignored or not. What do you think?

Reply:
Alternative approach from the hacking session:

 * show: boolean

By default all leaves would have `show: true` and branches `show: false`, but additional branches could be also assigned `show: true` or leaves hidden by `show: false`.

Reply:
@LecrisUT Will this changed approach work for you (and https://github.com/teemtee/fmf/issues/231) ?

Reply:
Indeed that's a cleaner solution. Only comment is if `show` is intuitive that it affects evaluation and not just `fmf show`/`tmt plans show`.

Reply:
Good point, while writing docs I've realized that `select` is probably more direct name.

Reply:
It is not immediately apparent, but after swirling it around in the head, it indeed becomes intuitive. I think it's just like the `inherit`, operators, etc., once you get in the mindset (what's the subject, result, etc.) it becomes intuitive

Reply:
Yeah, agreed, `select` sounds good.